### PR TITLE
grpc day blog post

### PR DESCRIPTION
**Pull request by YifeiZhuang**
[go/grpc-day-blog-post](http://goto.google.com/grpc-day-blog-post)

### CODE REVIEW

File: content/en/blog/grpc-day-2023.md
Comment (YifeiZhuang) [inline content/en/blog/grpc-day-2023.md:9]: @arvindbr8 when will the codelab be available? could you give me the link?

Comment (YifeiZhuang) [reply content/en/blog/grpc-day-2023.md:9]: blocked on cl/535292943

**Comment by easwars** [comment]
@YifeiZhuang : Do we still plan on merging this PR or should we close it?

**Comment by YifeiZhuang** [comment]
> @YifeiZhuang : Do we still plan on merging this PR or should we close it?

we still want to merge this PR after cl/535292943 is merged. 
@arvindbr8 do we have progress on cl/535292943? We want the codelab link.

**Comment by apolcyn** [comment]
@YifeiZhuang is this ready to submit?

**Comment by YifeiZhuang** [comment]
> @YifeiZhuang is this ready to submit?

@apolcyn thanks for ping! The code lab seems finally published! This is already reviewed and approved by Kevin and Neil, do you have the permission to submit? Please do if so.

**Comment by YifeiZhuang** [comment]
cc. @temawi  could you approve and merge?

**Review by temawi — APPROVED** [review]

**Pull request merged**
